scripts/package.json

```diff
@@ -138,11 +137,13 @@
     "knip": "^5.30.1",
     "lint-staged": "^15.2.7",
     "memoizerific": "^1.11.3",
+    "minimatch": "^10.0.1",
```

Comment:
I don't see usage of this anywhere in the repo

Reply:
The knip config file references it... I ran lint on scripts and this came up.